src/components/Tooltip/Tooltip.vue

```diff
@@ -48,6 +55,9 @@ export default {
 		},
 	},
 	computed: {
+		tooltipContent() {
+			return DOMPurify.sanitize(this.tooltip, sanitizeConfig);
```

Comment:
And here we can call the sanitizeHtml from the directive:
```suggestion
			return sanitizeHtml(this.tooltip);
```

then we can delete the `sanitizeConfig` variable.